dali/base/dautils.cpp

```diff
@@ -515,7 +523,17 @@ void CDfsLogicalFileName::set(const char *name, bool removeForeign)
         lfn.set(full);
         return;
     }
-    normalizeName(name, lfn, false);
+    if (isExternalFile(name))
+    {
+        lfn.set(name);
+        lfn.toLowerCase();
+        const char *ns1 = strstr(name,"::>");
+        if (ns1)
+            tailpos = ns1 - name + 2;
+        external = true;
+    }
+    else
+        normalizeName(name, lfn, false);
     if (removeForeign)
```

Comment:
This test/block can move into the else above now. i.e. it is not relevant to externals.